### PR TITLE
MAT-7587: Throw hqmServiceException

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7587](https://jira.cms.gov/browse/MAT-7587)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by ethankaplan — DISMISSED** [review]

File: src/main/java/gov/cms/madie/resources/CustomErrorHandlerAdvice.java [hunk 1]
```diff
@@ -31,6 +32,14 @@ Map<String, Object> handleCustomException(WebRequest request) {
     return getErrorAttributes(request, HttpStatus.BAD_REQUEST);
   }
 
+  // Handle HQMFServiceException separately
+  @ExceptionHandler(HQMFServiceException.class)
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ResponseBody
+  Map<String, Object> handleHQMFServiceException(WebRequest request) {
+    return getErrorAttributes(request, HttpStatus.INTERNAL_SERVER_ERROR);
```
Comment (adongare) [inline src/main/java/gov/cms/madie/resources/CustomErrorHandlerAdvice.java]: lil bit confusing to me:
line 37 -> HttpStatus.BAD_REQUEST
line 40 ->INTERNAL_SERVER_ERROR

Comment (adongare) [reply src/main/java/gov/cms/madie/resources/CustomErrorHandlerAdvice.java]: okay, after looking at this closely- we do not need to introduce new exception. All you have to do is to remove try catch in `src/main/java/gov/cms/madie/services/PackagingService.java` line 58

**Review by adongare — APPROVED** [review]
